resolwe_bio/tools/create_track_hubs.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-docstring,invalid-name
+"""Create UCSC track hub files."""
+
+import argparse
+import os
+
+
+def get_args():
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(description="Create files for a UCSC track hub")
+    parser.add_argument('--name', nargs='+', help="Sample names")
+    parser.add_argument('--track_hub_name', help="Track hub name")
```

Comment:
Use shorter names and `-` instead of `_`.

Reply:
Sorry, I proposed:
> Why not call this variable track_hub_name?

Domen is right, you should use `-`. I think the `--track-hub-name` is fine.

Reply:
I changed it to `--track` as I decided to use one word names.